### PR TITLE
Make setup.py files PyPI compatible

### PR DESCRIPTION
**Pull request by tiran**
Use correct classifiers and license

https://fedorahosted.org/freeipa/ticket/6484

Signed-off-by: Christian Heimes <cheimes@redhat.com>

### CODE REVIEW

File: ipasetup.py.in [hunk 1]
```diff
@@ -18,9 +17,10 @@
 import os
 import sys
 
+
 common_args = dict(
     version="@VERSION@",
-    license="GPL",
+    license="GPLv3+",
```
Comment (mbasti-rh) [inline ipasetup.py.in]: Shouldn't be this `GPLv3` only?

File: ipasetup.py.in [hunk 2]
```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/python2
-# Copyright (C) 2014  Red Hat
+# Copyright (C) 2016  Red Hat
```
Comment (mbasti-rh) [inline ipasetup.py.in]: Shouldn't be this in format 2014-2016?

File: ipasetup.py.in [hunk 3]
```diff
@@ -30,14 +30,21 @@ common_args = dict(
...
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: Implementation :: CPython",
         "Operating System :: POSIX",
+        "Operating System :: POSIX :: Linux",
         "Operating System :: Unix",
+        "Topic :: Internet :: Name Service (DNS)",
+        ("Topic :: System :: Systems Administration :: "
+         "Authentication/Directory :: LDAP"),
```
Comment (mbasti-rh) [inline ipasetup.py.in]: Maybe additional topic can be added as well:
- (maybe rather not include this one) Topic :: System :: Systems Administration :: Authentication/Directory :: NIS
- Topic :: Security
- Topic :: Internet :: WWW/HTTP :: WSGI :: Application

**Comment by mbasti-rh** [comment]
I have some inline questions

File: ipasetup.py.in (hunk 2 above)
Comment (tiran) [inline ipasetup.py.in]: The file was created 2016. I forgot to correct the year when I copied over the license header.

File: ipasetup.py.in (hunk 1 above)
Comment (tiran) [inline ipasetup.py.in]: Right

File: ipasetup.py.in (hunk 3 above)
Comment (tiran) [inline ipasetup.py.in]: None of the PyPI packages will contain a WSGI application. The WSGI is in ipaserver but we are not going to release the server on PyPI yet.

**Comment by pvoborni** [comment]
Fixed upstream
master:
https://fedorahosted.org/freeipa/changeset/2dd66c6366454f9edd9b89861530e97c75b2d869

**Pull request closed without merging**